flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -63,6 +73,14 @@ class FlxGamepad implements IFlxDestroyable
 	 * Helper class to get the justMoved, justReleased, and float values of analog input.
 	 */
 	public var analog(default, null):FlxGamepadAnalogList;
+	/**
+	 * Helper class to get the float values of motion-sensing input, if it is available
+	 */
+	public var motion(default, null):FlxGamepadMotionValueList;
+	/**
+	 * Helper class to get the float values of mouse-like pointer nput, if it is available
+	 */
+	public var pointer(default, null):FlxGamepadPointerValueList;
```

Comment:
I did? Is that what the blame history shows? Hmmm.

In any case, the values are not actually screen coordinates, they come off of the controller as two analog values between 0 and 1. Not only do they have no knowledge of flixel's coordinate system, they have no knowledge of the world coordinate systems either. It's just two values between the minimum and maximum values of the x/y axes of the detection range, whether it's from the sensor bar or the touchpad.

I mean I suppose I could map it world (OpenFL stage) coordinates using FlxPointer. 

Basically what I'm asking is, do you think there's any value in also reporting the 0-1 values, or will users just always multiply it times screen coordinates and be really annoyed that it doesn't do it for them.